DMCompiler/DM/DMProc.cs

```diff
@@ -134,6 +121,26 @@ private class DMProcScope {
             }
         }
 
+        public string Name => _astDefinition?.Name ?? "<init>";
+
+        private int AllocLocalVariable(string name) {
+            _localVariableNames.Add(name);
+            WriteLocalVariable(name);
+            return _localVariableIdCounter++;
+        }
+
+        private void DeallocLocalVariables(int amount) {
+            if (amount > 0) {
+                _localVariableNames.RemoveRange(_localVariableNames.Count - amount, amount);
+                WriteLocalVariableDealloc(amount);
+                _localVariableIdCounter -= amount;
+            }
+        }
+
+        public DreamPath GetPath() {
```

Comment:
Unused (along with confusingly getting the object's path rather than the proc's)